code/game/objects/structures/memorial.dm

```diff
@@ -13,4 +15,12 @@ This memorial has been designed for him and any future coders to perish.
 	icon = 'icons/obj/structures/memorial.dmi'
 	icon_state = "memorial"
 	density = TRUE
+	anchored = TRUE
+
+/obj/structure/prop/almayer/valmoric
+	name = "Tomb of the Unknown Soldier"
+	desc = "HERE RESTS IN\nHONORED GLORY\nAN SOLDIER\nKNOWN BUT TO GOD"
```

Comment:
`desc = "HERE RESTS IN\nHONORED GLORY\nA SOLDIER\nKNOWN BUT TO GOD"`
Needs to be "A Soldier," not "An Soldier."
Lovely work though. I like this a lot.